tests/ha/single_pod_restart/test_server_pod_restart.py

```diff
@@ -717,3 +718,169 @@ def test_reads_during_server_pod_restart(self):
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 8: Successfully ran IOs after server pod restart")
         LOGGER.info("ENDED: Test to verify READs during data pod restart.")
```

Comment:
not part of ur PR but this line need changes for server pod restart instead of data

Reply:
@rahul-hatwar  took care of this in PR#1803